web/src/components/profile/ProfileDateOfFormation.tsx

```diff
@@ -70,7 +70,7 @@ export const ProfileDateOfFormation = (props: Props): ReactElement => {
     process.env.NODE_ENV === "test" || process.env.CI === "true" ? DesktopDatePicker : DatePicker;
 
   return (
-    <LocalizationProvider dateAdapter={AdapterDayjs}>
+    <LocalizationProvider dateAdapter={AdapterDayjs} data-testid="profile-date-of-formation-question">
```

Comment:
Am I correct that this testId is never used? If so, it should be removed

Reply:
Yes, removed